govc/vm/create.go

```diff
@@ -519,6 +519,14 @@ func (cmd *create) addStorage(devices object.VirtualDeviceList) (object.VirtualD
 
 			devices = append(devices, nvme)
 			cmd.controller = devices.Name(nvme)
+		} else if cmd.controller == "sata" {
+			sata, err := devices.CreateSATAController()
```

Comment:
Just trying in the H5 UI for the first time..
`Add New Device -> Controllers -> SATA Controller`
Always creates a `VirtualAHCIController`, I don't see an option to create the base `VirtualSATAController` type.
Seems we should be doing the same, any insights on this?

```console
% govc device.info -vm my-vm ahci-*
Name:       ahci-15000
  Type:     VirtualAHCIController
  Label:    SATA controller 0
  Summary:  AHCI
  Key:      15000
  Devices:  
```

Reply:
Tried creating a vm with your branch:
```console
% govc vm.create -disk.controller sata -net "VM Network" -host $host -ds $ds sata-vm
govc: A specified parameter was not correct:  (Copying Virtual Machine configuration)
```
Works with this change:
```diff
modified   object/virtual_device_list.go
@@ -412,7 +413,7 @@ func (l VirtualDeviceList) FindSATAController(name string) (types.BaseVirtualCon
 
 // CreateSATAController creates a new SATA controller.
 func (l VirtualDeviceList) CreateSATAController() (types.BaseVirtualDevice, error) {
-	sata := &types.VirtualSATAController{}
+	sata := &types.VirtualAHCIController{}
 	sata.BusNumber = l.newSATABusNumber()
 	sata.Key = l.NewKey()
```

Reply:
Resolved in 2e77836.